### PR TITLE
fix/unblock  analytics tests + move create_alias event to 'enter wallet' button click

### PR DESCRIPTION
**Pull request by oldGreg5**
Unblock tests LW-8311,LW-7365 [here](https://input-output.atlassian.net/browse/LW-10488).
After discussion with @vhulchenko-iohk and @szymonmaslowski I moved create_event sending to 'enter wallet' button click
Also new bug popped up during this [LW-10635](https://input-output.atlassian.net/browse/LW-10635) and will be fixed here as well

[LW-10635]: https://input-output.atlassian.net/browse/LW-10635?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by szymonmaslowski — APPROVED** [review]

File: packages/e2e-tests/src/features/analytics/AnalyticsOnboardingEvents.feature [hunk 1]
```diff
@@ -28,16 +28,18 @@ Feature: Analytics - Posthog - Onboarding - Extended View
     And I click "Enter wallet" button
     Then I validate latest analytics multiple events:
       | onboarding \| restore wallet revamp \| let's set up your new wallet \| enter wallet \| click |
+      | onboarding \| restore wallet revamp \| added                                                 |
+      | $merge_dangerously                                                                           |
       | $create_alias                                                                                |
```
Comment (bslabiak) [inline packages/e2e-tests/src/features/analytics/AnalyticsOnboardingEvents.feature:33]: Should we sent $merge_dangerously event on normal onboarding? I tough it is just for multiwallet feature when we merging 2 wallets together.

Comment (szymonmaslowski) [reply packages/e2e-tests/src/features/analytics/AnalyticsOnboardingEvents.feature:33]: I didn't know that. Then this is a bug 😉

Comment (oldGreg5) [reply packages/e2e-tests/src/features/analytics/AnalyticsOnboardingEvents.feature]: Bug registered LW-10635 but it's simple change and will be fixed by this PR

File: apps/browser-extension-wallet/src/views/browser-view/features/multi-wallet/hardware-wallet/context.tsx [hunk 2]
```diff
@@ -202,7 +202,10 @@ export const HardwareWalletProvider = ({ children }: HardwareWalletProviderProps
       $set: { wallet_accounts_quantity: await getWalletAccountsQtyString(walletRepository) }
       /* eslint-enable camelcase */
     });
-    await analytics.sendMergeEvent(cardanoWallet.source.account.extendedAccountPublicKey);
+
+    if (mergeEventRequired) {
+      await analytics.sendMergeEvent(cardanoWallet.source.account.extendedAccountPublicKey);
+    }
```
Comment (szymonmaslowski) [inline apps/browser-extension-wallet/src/views/browser-view/features/multi-wallet/hardware-wallet/context.tsx:208]: Please add `mergeEventRequired` to the array in line 219

**Review by szymonmaslowski — CHANGES_REQUESTED** [review]

File: apps/browser-extension-wallet/src/views/browser-view/features/multi-wallet/hardware-wallet/context.tsx [hunk 3]
```diff
@@ -202,7 +202,10 @@ export const HardwareWalletProvider = ({ children }: HardwareWalletProviderProps
       $set: { wallet_accounts_quantity: await getWalletAccountsQtyString(walletRepository) }
       /* eslint-enable camelcase */
     });
-    await analytics.sendMergeEvent(cardanoWallet.source.account.extendedAccountPublicKey);
+
+    if (mergeEventRequired) {
```
Comment (szymonmaslowski) [inline apps/browser-extension-wallet/src/views/browser-view/features/multi-wallet/hardware-wallet/context.tsx:206]: Could we please keep the order of events in the HW and restore/create flows (useHotWalletCretion) the same? In the restore/create we send them in the order of: alias, merge, hd, but in here HW it is: merge, hd, alias. Should we change the order here to the same as in the restore/create?

**Review by szymonmaslowski — CHANGES_REQUESTED** [review]

**Review by szymonmaslowski — APPROVED** [review]

**Pull request merged**